docs/src/pages/system/typography/LineHeight.js

```diff
@@ -9,7 +9,7 @@ export default function LineHeight() {
         Normal height.
       </Box>
       <Box lineHeight={10} m={1}>
-        10 px.
+        line-height: 10
```

Comment:
```suggestion
        line-height: 2
```

Reply:
2 seems much more "sane". Thanks for reviewing.